bitnami/elasticsearch/templates/ingest/pdb.yaml

```diff
@@ -0,0 +1,23 @@
+{{- if and (include "elasticsearch.ingest.enabled" .) .Values.ingest.pdb.create }}
```

Comment:
This part would be more correct to write it as follows:
`{{- if and ( include "elasticsearch.ingest.enabled" . ) .Values.ingest.pdb.create }}`
This change should be made to all pdb.yaml files in this PR.